scripts/migrate_assets.js

```diff
@@ -25,56 +25,24 @@
  * ENHANCEMENTS, OR MODIFICATIONS.
  */
 
-var _ = require('lodash');
-var async = require('async');
-var contentDisposition = require('content-disposition');
-var fs = require('fs');
-var os = require('os');
-var fs = require('fs');
-var path = require('path');
-var request = require('request');
 var argv = require('yargs')
-    .usage('Usage: $0 --fromcourse [fromcourse] --fromuser [fromuser] --tocourse [tocourse] --touser [touser] --categories [categories]')
-    .demand(['fromcourse', 'fromuser', 'tocourse', 'touser'])
```

Comment:
Because a SuiteC user account is associated with only one course, the `fromcourse` and `tocourse` arguments are redundant.